pkg/file/io.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+package file
+
+import (
+	"bytes"
+
+	"io"
+)
+
+// simpleJoinerReadCloser wraps a byte slice in a io.ReadCloser implementation.
+type simpleReadCloser struct {
+	buffer io.Reader
+}
+
+func NewSimpleReadCloser(buffer []byte) io.ReadCloser {
+	return &simpleReadCloser{
+		buffer: bytes.NewBuffer(buffer),
+	}
+}
+
+// Read implements io.Reader.
+func (s *simpleReadCloser) Read(b []byte) (int, error) {
+	return s.buffer.Read(b)
+}
+
+// Close implements io.Closer.
+func (s *simpleReadCloser) Close() error {
+	return nil
```

Comment:
i am aware that this is supposed to be a `simple` read closer, but i am wondering if we should allow any reads after a `Close` has been called